student_explorer/templates/403.html

```diff
@@ -7,6 +7,6 @@
 {% block content %}
 <div class="container">
     <h1>Permission Denied</h1>
-    <p>You do not currently have access to Student Explorer, an early-warning tool that uses course and assignment data from Canvas to help advisors, coaches, and mentors identify at-risk students and facilitate outreach to these students. For more information about Student Explorer including how to get access and who to contact for assistance, please see the <a href="https://documentation.its.umich.edu/node/831">Student Explorer Documentation.</a></p>
+    <p>You do not currently have access to Student Explorer. To learn about Student Explorer and its history, see <a href="/about">About Student Explorer</a>. For more information about using Student Explorer including how to get access and who to contact for assistance, please see the <a href="https://documentation.its.umich.edu/student-explorer-general">Student Explorer documentation.</a></p>
```

Comment:
I might add a comma to these so that it's "For more information about Student Explorer, including ..."

A little jarring to have one imperative with please and one without ("see About Student Explorer", "please see the Student Explorer documentation"), but it's not a big deal.

Reply:
Agree with both. I'll make those changes